app/src/main/res/layout/app_bar.xml

```diff
@@ -45,15 +45,15 @@
         <View
             android:layout_width="1dp"
             android:layout_height="match_parent"
-            android:background="?attr/colorPrimary" />
+            android:background="?attr/white" />
 
         <ImageButton
             android:id="@+id/action_user"
             android:layout_width="0dp"
             android:layout_height="match_parent"
             android:layout_margin="10dp"
             android:layout_weight="1"
-            android:background="@color/white"
+            android:background="@color/colorPrimary"
```

Comment:
Same as above, this will make the icon to be invisible because of same color with background. You can just leave it as unchanged